<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="ImageManager" name="imageManager" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="ReflectanceMap" name="reflectanceMap" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="actionLoad_BSE_images"/>
    <addaction name="actionLoad_reflectance_maps"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionShow_normal_image"/>
   </widget>
   <widget class="QMenu" name="menuDetector">
    <property name="title">
     <string>Detector</string>
    </property>
    <addaction name="actionPort_angle"/>
   </widget>
   <addaction name="menuLoad"/>
   <addaction name="menuShow"/>
   <addaction name="menuDetector"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_BSE_images">
   <property name="text">
    <string>Load BSE images</string>
   </property>
  </action>
  <action name="actionLoad_reflectance_maps">
   <property name="text">
    <string>Load reflectance maps</string>
   </property>
  </action>
  <action name="actionShow_normal_image">
   <property name="text">
    <string>Show normal image</string>
   </property>
  </action>
  <action name="actionPort_angle">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ReflectanceMap</class>
   <extends>QWidget</extends>
   <header>../ReflectanceMap/ReflectanceMap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageManager</class>
   <extends>QWidget</extends>
   <header>../ImageManager/ImageManager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
